apisix/plugins/limit-req.lua

```diff
@@ -67,7 +67,17 @@ function _M.access(conf, ctx)
         return 500
     end
 
-    local key = (ctx.var[conf.key] or "") .. ctx.conf_type .. ctx.conf_version
+    local key
+    if conf.key == "consumer_name" then
+        if not ctx.consumer_id then
+            core.log.error("Missing consumer's username.")
```

Comment:
still not fix

Reply:
take a look: https://github.com/apache/apisix/pull/2270#discussion_r494019356

Reply:
sorry, it's fixed now.